supervisord.conf

```diff
@@ -0,0 +1,35 @@
+[supervisord]
+nodaemon=true
+loglevel=warn
+logfile_maxbytes=0
+logfile=/dev/stderr
+
+[program:nginx]
+command = nginx -g 'daemon off;'
+autostart = true
+stdout_logfile=/dev/stdout
+stdout_logfile_maxbytes=0
+stderr_logfile=/dev/stderr
+stderr_logfile_maxbytes=0
+
+[program:php-fpm7.3]
+command = /usr/sbin/php-fpm7.3 -F
+autostart = true
+stdout_logfile=/dev/stdout
+stdout_logfile_maxbytes=0
+stderr_logfile=/dev/stderr
+stderr_logfile_maxbytes=0
+
+[program:foo]
+directory = /var/www/app/wordpress
+command = sh -c 'sleep 7; ./wp core install --url="localhost" --title="Blogging Platform" --admin_user=admin --admin_email=danpal@example.com --admin_password="password" --allow-root; sleep 4; ./wp theme install twentynineteen --activate --allow-root; sleep 4; ./wp plugin install %(ENV_PLUGIN_URL)s --activate --allow-root'
```

Comment:
can you make those parameters as variables:
- blog url
- blog title
- admin user
- admin password
- admin email
- theme name

Why do you need sleep 4 after every programm?